profile/E-ARK-eHealth1-REPRESENTATION.xml

```diff
@@ -595,6 +599,7 @@
             </requirement>
             <requirement ID="EH39" REQLEVEL="SHOULD" EXAMPLES="structMapExample2">
                 <description>
+                <!--Example does not show documentation struct in use, applies to all reqs through eh43-->

```

Comment:
Example is somewhat lacking and needs updating. Note that EH43 is depracated in v2.0.0.